README.md

```diff
@@ -141,17 +141,19 @@ The STL uses boost-math headers to provide P0226R1 Mathematical Special Function
 acquire this dependency.
 
 1. Install Visual Studio 2019 16.6 Preview 2 or later.
-2. Invoke `git clone https://github.com/microsoft/vcpkg`
-3. Invoke `cd vcpkg`
-4. Invoke `.\bootstrap-vcpkg.bat`
-5. Assuming you are targeting x86 and x64, invoke `.\vcpkg.exe install boost-math:x86-windows boost-math:x64-windows`
+2. Change directories to a location where you'd like a clone of this STL repository.

```

Comment:
This is substantially duplicating the command-line instructions for initial setup, although it eventually uses the IDE for building. Is it possible to use the IDE to clone the repo and init the vcpkg submodule, so IDE users can remain in the IDE as much as possible?

Reply:
From my test when I provided my first comment, cloning with VS already did that for both submodules.